pallets/messages/src/benchmarking.rs

```diff
@@ -1,61 +1,83 @@
+#![cfg(feature = "runtime-benchmarks")]
```

Comment:
why do you need this if you are only including it conditionally in messages/src/lib.rs
```

#[cfg(feature = "runtime-benchmarks")]
mod benchmarking;
```

Reply:
oh yeah. your right.